src/pkg/caendr/caendr/models/run/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Parent classes
+from .base   import Runner
+from .gcp    import GCPRunner, GCPCloudRunRunner, GCPLifesciencesRunner
```

Comment:
Unless we still need LifeSciences logic, we should plan on retiring all references to it.

Reply:
Agreed. I was hesitant to delete it entirely, so I've tried to modularize it as much as possible and remove remaining imports.

I removed one of the imports in [f7000d5](https://github.com/AndersenLab/CAENDR/pull/404/commits/f7000d5ddc10d0a41cd49606718b93bb61510c40).

There are a few remaining references I could find:
- The dedicated `lifesciences.py` files, in `services/cloud/` and in `models/`
- The `PipelineRunError` class in `error.py` (which holds almost every error in the app, I think)
- The `cloud/utils.py` function `get_operation_status`, which routes the a status check to the correct function (lifesciences or cloudrun, in our case)
- The `GCPLifesciencesRunner` class.  This is marked as deprecated, and intentionally doesn't implement all of its parent's abstract methods, so in practice it can't be instantiated.

I'm happy to move `GCPLifesciencesRunner` to its own file and write "deprecated" everywhere, but I'd like to do that in a new branch if possible -- I have some incoming changes to the `Runner`s that I'd like to PR first, and I'd like to avoid merge conflicts.